message_app_client/src/components/conversation.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+class ConversationCard extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            lastMessageContent: null,
+            conversationStatus: null,
+            title: null,
+            conversationID: null,
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+            lastMessageContent: this.props.lastMessageContent,
+            conversationStatus: this.props.conversationStatus,
+            title: this.props.title,
+            conversationID: this.props.conversationID
+        });
+    }
+
+    componentDidUpdate(prevProps){
+        // Update the state when the conversation has a new lastMessage
+        if (this.props.lastMessageContent !== prevProps.lastMessageContent){
+            this.setState({
+                lastMessageContent: this.props.lastMessageContent
+            });
+        }
+
+        // Update the state when the conversation has a new status
+        if (this.props.conversationStatus !== prevProps.conversationStatus){
+            this.setState({
+                conversationStatus: this.props.conversationStatus
+            });
```

Comment:
You might need to update/change this based on the comment above about the "status" of a conversation.